app/templates/views/unsubscribe-request-report.html

```diff
@@ -0,0 +1,37 @@
+{% extends "withnav_template.html" %}
+{% from "components/form.html" import form_wrapper %}
+{% from "components/page-header.html" import page_header %}
+{% from "govuk_frontend_jinja/components/back-link/macro.html" import govukBackLink %}
+{% from "govuk_frontend_jinja/components/button/macro.html" import govukButton %}
+
+{% block service_page_title %}
+{{ earliest_timestamp|format_date_normal }} until {{ latest_timestamp|format_date_normal }}
+{% endblock %}
+
+
+{% block backLink %}
+  {{ govukBackLink({ "href": url_for('main.unsubscribe_request_reports_summary', service_id=current_service.id) }) }}
+{% endblock %}
+
+{% block maincolumn_content %}
+
+    {{ page_header('{} until {}'.format(earliest_timestamp|format_date_normal, latest_timestamp|format_date_normal)) }}
+
+ <p class="govuk-body">
+     {{ count|format_thousands}} new requests to unsubscribe
+ </p>
+
+    <p class="govuk-body">You must: </p>
+      <ol class="govuk-list govuk-list--number">
+        <li><span class="bottom-gutter"><a download href="" class="govuk-link heading-small">Download the report </a>
+          &nbsp(available for 90 days) </span> </li>
```

Comment:
If you really want a non-breaking space here it needs to be encoded like this with no other whitespace around it:
```suggestion
        <li><span class="bottom-gutter"><a download href="" class="govuk-link heading-small">Download the report</a>&nbsp;(available for 90 days) </span> </li>
```
Otherwise:
```suggestion
        <li><span class="bottom-gutter"><a download href="" class="govuk-link heading-small">Download the report </a>
          (available for 90 days) </span> </li>
```

Reply:
I think it looks better with a non breaking space. I will go with your suggestion to remove the whitespace.